### PR TITLE
PlayerZero Web Docs (Deploy)

### PR DESCRIPTION
**Pull request by markzegarelli**
- Docs Commit for PlayerZero Web
- Insert actions block

### Related issues (optional)
Duplicates #4135

### CODE REVIEW

**Comment by AnimeshKoratana** [comment]
Thanks @markzegarelli 🙌 ! 
I missed the actions block part

File: src/connections/destinations/catalog/actions-playerzero-web/index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+[PlayerZero](https://www.playerzero.app/){:target="_blank"} is an application monitoring platform 
+that measures the impact of engineering issues on important product outcomes (for example: successful onboardings, checkouts, or conversions).
+
+The Segment integration for PlayerZero automatically instruments your web application to send engineering and product analytics events to PlayerZero.
+
+## Getting started
+
+1. From the Segment web app, click **Catalog**, then click **Destinations**.
+2. Find the Destinations Actions item in the left navigation, and click it.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-playerzero-web/index.md:17]:
```suggestion
2. Under **Categories** select **Destination Actions**.
```

**Review by stayseesong — CHANGES_REQUESTED** [review]

**Comment by AnimeshKoratana** [comment]
These changes look great, thanks @stayseesong
@markzegarelli can you accept them? I think since you're the PR author, I can't accept the suggested changes.

Also happy to go in and make the changes directly on the file itself if faster.

**Comment by AnimeshKoratana** [comment]
@stayseesong @markzegarelli is there anything else to be done on our end before the deploy?

**Review by stayseesong — APPROVED** [review]

**Pull request merged**